src/icons/fileIcons.ts

```diff
@@ -2570,5 +2570,10 @@ export const fileIcons: FileIcons = {
         '.k8s.yaml',
       ],
     },
+    {
+      name: 'interface',
+      fileNames: ['interface.ts', 'interfaces.ts'],
```

Comment:
I'd like to keep these typescript files without special icons. I want to avoid that the extension will have too many separations between "normal" typescript files. I see files with the pattern "interface.ts" as normal typescript files which should have the standard typescript logo. Having a more specific file icon will make it harder to recognize which files are related to TypeScript and which not.